test/coap_message_encode_decode_test.dart

```diff
@@ -564,16 +564,16 @@ void main() {
           48,
           57,
           202,
+          110,
+          105,
+          97,
+          108,
+          112,
+          47,
           116,
-          101,
           120,
+          101,
           116,
-          47,
-          112,
-          108,
-          97,
-          105,
-          110,
```

Comment:
It took me a couple of minutes but I now realized that the behavior of this test changed because the content-format is encoded as a string value and not as a the numeric representation from the registry, hence the mixed up bytes. I think we should open an issue for this or mention it in #49 as changing the encoded value to a numeric one messes up the tests for the older drafts :/ After the old drafts have been removed we could start fixing/updating the tests as a whole.